internal/csi-addons/rbd/replication.go

```diff
@@ -794,6 +805,68 @@ func getLastSyncTime(description string) (*timestamppb.Timestamp, error) {
 	return lastSyncTime, nil
 }
 
+// This function gets the last snapshot bytes from the description
+// of localStatus and converts it into required type.
+func getLastSyncBytes(description string) (int64, error) {
+	// Format of the description will be as followed:
+	// description = `replaying, {"bytes_per_second":0.0,"bytes_per_snapshot":81920.0,
+	//"last_snapshot_bytes":81920,"last_snapshot_sync_seconds":0,"local_snapshot_timestamp":1684675261,
+	//"remote_snapshot_timestamp":1684675261,"replay_state":"idle"}`
+	// In case there is no last snapshot bytes returns 0 as the
+	// LastSyncBytes is optional.
+	if description == "" {
+		return 0, nil
+	}
+	splittedString := strings.SplitN(description, ",", 2)
+	if len(splittedString) == 1 {
+		return 0, nil
+	}
+	type localStatus struct {
+		LocalSnapshotBytes int64 `json:"last_snapshot_bytes"`
+	}
```

Comment:
This code is repeated for `getLastSyncDuration()`, please put it in a helper function that returns a `localStatus`.